pkg/resmgr/control/cpu/cpu.go

```diff
@@ -123,22 +122,6 @@ func (ctl *cpuctl) PostStopHook(c cache.Container) error {
 	return nil
 }
 
-// enforceCpufreqGovernor enforces a class-specific cpufreq governor to a cpuset.
-func (ctl *cpuctl) enforceCpufreqGovernor(class string, cpusIds ...int) error {
-	if _, ok := ctl.classes[class]; !ok {
-		return fmt.Errorf("non-existent cpu class %q", class)
-	}
-	governor := ctl.classes[class].FreqGovernor
-	for cpu := range cpusIds {
-		log.Info(strconv.Itoa(cpu), governor)
-	}
-	log.Debug("enforcing cpu frequency governor %q on %v", governor, cpusIds)
-	if err := utils.SetScalingGovernorForCPUs(cpusIds, governor); err != nil {
-		return fmt.Errorf("Cannot set cpufreq governor %d: %w", governor, err)
-	}
-	return nil
-}
-
 // enforceCpufreq enforces a class-specific cpufreq configuration to a cpuset
 func (ctl *cpuctl) enforceCpufreq(class string, cpus ...int) error {
 	if _, ok := ctl.classes[class]; !ok {
```

Comment:
Since we're now using `ctl.classes[class]` quite a bit in this function, cannot we rewrite the sentinel-test to take, keep and use the looked up class if it exists ? So something like

```golang
...
    c, ok := ctl.classes[class]
    if !ok {
        return fmt.Errorf("non-existent cpu class %q", class)
    }

    min, max := int(c.MinFreq), int(c.MaxFreq)
    ...

    if governor = c.FreqGovernor; governor != "" {
        ...
    }

    ...
```